test/conftest.py

```diff
@@ -159,17 +210,530 @@ def scratch_dir(config: dict[str, str]) -> str:
 
 
 @pytest.fixture(scope="session")
-def genome_refs(clients: dict[str, Any], ws_id: str) -> list[str]:
-    return make_genome_refs(clients["foft"], ws_id)
+def dummy_shock_file_handle(clients: dict[str, Any], scratch_dir: str) -> str:
+    """Upload a (tiny!) file to shock and return the resulting file.
+
+    :param clients: clients dictionary
+    :type clients: dict[str, Any]
+    :param scratch_dir: scratch directory containing the dummy file
+    :type scratch_dir: str
+    :return: shock handle for the uploaded file
+    :rtype: str
+    """
+    # "dummy.txt" lives in the test/data/ directory and is copied to the scratch dir during test set up
+    dummy_file_info = clients["dfu"].file_to_shock(
+        {"file_path": os.path.join(scratch_dir, "dummy.txt"), "make_handle": 1}
+    )
+    return dummy_file_info["handle"]
+
+
+@pytest.fixture(scope="session")
+def alignment_refs(
+    clients: dict[str, Any],
+    ws_id: int,
+    genome_refs: list[str],
+    reads_refs: list[str],
+    dummy_shock_file_handle: str,
+) -> list[str]:
+    """Create some KBase alignment objects and return the refs.
+
+    :param clients: clients dictionary
+    :type clients: dict[str, Any]
+    :param ws_id: workspace ID
+    :type ws_id: int
+    :param genome_refs: genome refs (see fixture)
+    :type genome_refs: list[str]
+    :param reads_refs: reads refs (see fixture)
+    :type reads_refs: list[str]
+    :param dummy_shock_file_handle: handle for dummy upload file
+    :type dummy_shock_file_handle: str
+    :return: list of KBase UPAs
+    :rtype: list[str]
+    """
+    return [
+        make_fake_alignment(
+            dummy_shock_file_handle,
+            f"fake_alignment_{idx}",
+            reads_ref,
+            genome_refs[0],
+            ws_id,
+            clients["ws"],
+        )
+        for idx, reads_ref in enumerate(reads_refs)
+    ]
 
 
 @pytest.fixture(scope="session")
-def reads_refs(clients: dict[str, Any], ws_id: str) -> list[str]:
-    return make_reads_refs(clients["foft"], ws_id)
+def mismatched_alignment_refs(
```

Comment:
Oh... maybe not quite alphabetical order. Alphabetical apart from these mismatched refs.